examples/tool.xml

```diff
@@ -8,20 +8,23 @@
   <stdio>
     <exit_code range="1:" level="fatal"/>
   </stdio>
+  <expand macro="stdio"/>
   <version_command><![CDATA[aragorn.exe --version]]></version_command>
   <command><![CDATA[aragorn.exe $flag
--float $float
---float-fromarg $float_fromarg
-#if str($Select) == "bye"
--some_int $some_int
+-float $float_section.float
+--float-fromarg $float_section.float_fromarg
+#if str($cond.Select) == "bye"
+-some_int $cond.some_int
 #end if
 -i$int_min,$int_max
 
 $posint
 select_local $select_local
 #for $i in $repeat:
---data $data
-#end for]]></command>
+--data $repeat.data
```

Comment:
is this expected? (haven't looked at the repeat code in a while.)

Reply:
Right, this should be `$i.data`